vector/src/main/java/im/vector/adapters/RoomViewHolder.java

```diff
@@ -195,6 +197,28 @@ public void populateViews(final Context context,
             vRoomName.setText(displayName);
         }
 
+        if (null != vRoomType) {
```

Comment:
Add a tool function that returns the room type. This function should be used in multiple cases.
Possible room types : PRIVATE, UNRESTRICTED or EXTERN, FORUM, DISCUSSION, UNKNOWN

Reply:
I agree, I will do it later